tests/publiccloud/run_ltp.pm

```diff
@@ -141,12 +141,14 @@ sub cleanup {
 
 sub process_ltp_known_issues {
     my ($self, $ltp_log) = @_;
-    my $decoded_ltp_log = Mojo::JSON::from_json($ltp_log);
-    my $env             = gen_ltp_env();
+    my $content         = Mojo::File::path($ltp_log)->slurp;
+    my $decoded_ltp_log = Mojo::JSON::from_json($content);
+    my $env             = $self->gen_ltp_env();
     my $suite           = get_required_var('COMMAND_FILE');
 
     foreach my $test (@{$decoded_ltp_log->{results}}) {
         if (($test->{status} eq 'fail' || $test->{status} eq 'brok' || $test->{status} eq 'warn')) {
+            $env->{retval} = $test->{retval};
             my $entry = find_whitelist_entry($env, $suite, $test->{test_fqn});
```

Comment:
```suggestion
            $env->{retval} = $test->{retval};
            my $entry = find_whitelist_entry($env, $suite, $test->{test_fqn});
            next unless (defined($entry));
```

Reply:
We never check if a whitelist entry exists, so from my perspective we just blindly overwrite all results?!